packages/blockchain-wallet-v4-frontend/src/data/analytics/model.js

```diff
@@ -1,6 +1,17 @@
-export const CUSTOM_DIMENSIONS = {
-  CURRENCY_PREFERENCE: 1
+//
+// CUSTOM VARIABLES
+//
+export const CUSTOM_VARIABLES = {
+  KYC_TIER: {
+    ID: 1,
+    NAME: 'kyc_tier'
```

Comment:
BE doesn't know the user tiers?

Reply:
we want to use it to be able to easily setup funnels and goals based on this data